packages/office-ui-fabric-react/src/components/Autofill/Autofill.tsx

```diff
@@ -75,9 +75,15 @@ export class Autofill extends BaseComponent<IAutofillProps, IAutofillState> impl
 
   public componentDidUpdate() {
     const value = this._value;
-    const { suggestedDisplayValue, shouldSelectFullInputValueInComponentDidUpdate } = this.props;
+    const { suggestedDisplayValue, shouldSelectFullInputValueInComponentDidUpdate, shouldSelectValueInComponentDidUpdate } = this.props;
     let differenceIndex = 0;
 
+    // We need to explicitly not select the text in the autofill if we are no longer focused. In IE11, selecting
+    // a input will also focus the input, causing other element's focus to be stolen.
+    if (shouldSelectValueInComponentDidUpdate && !shouldSelectValueInComponentDidUpdate()) {
```

Comment:
If the return value of this function was inverted then you could simplify this if check to be one statement

Reply:
I don't think so, we still have to check if the prop function exists before we can call it, however we can do this if we change this from a () -> bool to a bool